web_src/css/runner.css

```diff
@@ -52,3 +52,10 @@
   background-color: var(--color-yellow);
   color: var(--color-white);
 }
+
+.runner-container .runner-tags {
+    display: flex;
```

Comment:
It doesn't look good to me to change a `td`'s "display" like this.

Reply:
yes you're right; it wasn't really adding anything to the implementation so it has been removed in the latest commit